ratom/score.go

```diff
@@ -248,12 +287,17 @@ func Analyze(url string, client *http.Client) Module {
 		busFactor = -1.0
 		responsiveMaintainer = -1.0
 		license = false
+		versionPinning_score = -1.0
+		codeReviews_score = -1
```

Comment:
Since this is also float32, should be -1.0 to match others and avoid confusion
```suggestion
		codeReviews_score = -1.0
```